frontend/src/components/Sidebar/Sidebar.tsx

```diff
@@ -39,8 +53,20 @@ const Sidebar: FunctionComponent = ({ children }) => {
             </Link>
           </div>
         ))}
+        <div className={styles.logout}>
+          <img alt="profile_picture" src={`https://${profile}`} />
```

Comment:
Since `photoLink` might not always be defined, you may want to use a default icon when `photoLink` is not defined otherwise it will look like this:
![image](https://user-images.githubusercontent.com/48143149/112737007-91f53a80-8f2d-11eb-96c1-83700b947620.png)

Reply:
Is there a certain user icon that we currently have available? or should i use a dummy user profile image from the internet?

Reply:
Not sure if there is one so using a dummy image should be fine.